README.md

```diff
@@ -14,11 +14,11 @@ Head over to [HacktoberFest](https://hacktoberfest.digitalocean.com/), sign up a
 - Add/append your details into the `README.md` in the format:
 
 ```
-- [Name](link_to_github)
+- <br />[Name](link_to_github)
```

Comment:
Don't use break Tags, It's a markdown file, leave an empty line in between for a newline.

Reply:
Okk.

Reply:
Markdown does support break tags I guess😅